cmd/soroban-rpc/internal/daemon/daemon.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"errors"
 	"net/http"
+	_ "net/http/pprof"
```

Comment:
It would be nice if we could avoid the automatic http handler installed by this... :/

Reply:
It won't change until Go 2 at least, https://github.com/golang/go/issues/42834

Reply:
Problem is that the default mux code isn't exposed so I would need to replicate it privately

Reply:
since we're using a custom server anyway ( i.e. http.Server ), couldn't we use a non-default mux ?

Reply:
We could use a non-default mux but the default mux would still be filled automatically. See https://cs.opensource.google/go/go/+/refs/tags/go1.20.3:src/net/http/pprof/pprof.go;l=85-91